features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml.j2

```diff
@@ -20,49 +20,70 @@
 <APIResourceCollections>
     {% for collection in api_resource_collections %}
     <APIResourceCollection name="{{ collection.name }}" displayName="{{ collection.displayName }}" type="{{ collection.type | default('Unknown') }}" enabled="{{ collection.enabled | default('true') }}">
-        <Scopes>
-            {% if collection.scopes.feature %}
+        {% for scopes in collection.scopes %}
+        <Scopes version="{{ scopes.version }}">
```

Comment:
Can't we hard code as v1?

Reply:
Yes since we are not planning to future version, no need to template this neh?

Reply:
+1